napari/_tests/test_adding_removing.py

```diff
@@ -120,7 +121,7 @@ def my_custom_callback():
     for em in layer.events.emitters.values():
         # warningEmitters are not connected when connecting to the emitterGroup
         if not isinstance(em, WarningEmitter):
-            assert len(em.callbacks) == 1
+            assert len(em.callbacks) == count_warning_events(em.callbacks) + 1
```

Comment:
This is talked about [here](https://github.com/napari/napari/pull/5861/files#r1289480977)